packages/core/tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
     "useUnknownInCatchVariables": false,
-    "types": ["node", "vite/client"]
+    "types": ["node", "vite/client", "bun"]
```

Comment:
Reviewer note: Ideally, I would love for this types file to _only_ apply to `packages/core/src/adapters/bun.ts` and _nothing_ else, but I don't know how to do that. For now, it applies globally—if anybody knows how to scope it that'd be awesome otherwise 🤷 it is what it is.

Reply:
Can do `/// <reference types="bun" />`

Reply:
That worked! Yes!